maas-api/deploy/README.md

```
@@ -0,0 +1,32 @@
+├── overlays      <4>
+│   ├── dev
+│   ├── odh
+│   └── secret
+├── policies      <5>
+└── rbac          <6>
+```
+
+**<1> base** - Core MaaS API deployment manifests (service, deployment) with common labels and RBAC
+
+**<2> infra** - Infrastructure dependencies for Gateway API, Kuadrant, and OpenDataHub integration
+  * `<*>` - ODH minimal deployment to support models (`LLMInferenceService` machinery)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)